src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -31,41 +32,74 @@ const MainNavDroplist = Droplist.extend({
         <img class="app-nav__header-logo" src="/rwell-logo.svg" />
       {{else}}
         <div class="u-text--overflow">
-          <h2 class="app-nav__header-title u-text--overflow">{{ orgName }}</h2>
+          <h2 class="app-nav__header-title u-text--overflow">{{ workspaceName }}</h2>
           <span class="app-nav__header-arrow">{{far "angle-down"}}</span>
         </div>
         <div class="u-text--overflow">{{ userName }}</div>
       {{/if}}
     `,
     templateContext() {
       const currentUser = Radio.request('bootstrap', 'currentUser');
-      const currentOrg = Radio.request('bootstrap', 'organization');
+      const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
 
       return {
         userName: currentUser.get('name'),
-        orgName: currentOrg.get('name'),
+        workspaceName: currentWorkspace.get('name'),
         isMinimized: this.getOption('state').get('isMinimized'),
       };
     },
   },
   picklistOptions() {
     return {
       className: 'picklist app-nav__picklist',
```

Comment:
I think this can be simplified a bit by passing a template here:
something like:
```js
template: hbs`
    <div class="app-nav__picklist-heading">{{ @intl.globals.appNav.appNavViews.mainNavDroplist.organizationHeading }}</div>
    </div>
    <div class="app-nav__picklist-workspace-name">{{ name }}</div>
    <ul class="flex-region picklist__scroll js-picklist-scroll"></ul>
`
```
Then it seems like `headingClassName`, `itemClassName`, and `itemTemplate` could probably be defined here as well rather than on each list.

Which I think allows for:
```js
const currentOrg...
const workspacesMenu....
this.showChildView('navMain', new MainNavDroplist({
  model: currentOrg,
  lists: [{ collection: workspacesMenu }, { collection: appNavMenu }], 
  state: this.getState(),
}));
```

Reply:
Although.. I'm now thinking.. why not include the help and sign out links in the template?
Otherwise we have to deal with some items changing the selected state and some not..  then all we're passing is a collection of workspaces.

Reply:
Updated to do this in the last push.

But ran into an issue with the help and sign out event trigger functions. I'll make comments on those portions of the code.